src/lib/Microsoft.Health.Fhir.Ingest/Service/NormalizationEventConsumerService.cs

```diff
@@ -189,30 +188,34 @@ private void ProcessEvent(IEventMessage evt, IContentTemplate template, IList<(s
 
         private void NormalizeMessage(JObject token, IEventMessage evt, IContentTemplate template, IList<(string sourcePartition, IMeasurement measurement)> collector)
         {
-            Stopwatch sw = Stopwatch.StartNew();
             int projections = 0;
 
-            foreach (var measurement in template.GetMeasurements(token))
+            using (ITimed normalizeDuration = _logger.TrackDuration(IomtMetrics.NormalizedEventGenerationTimeMs(evt.PartitionId)))
             {
-                try
+                foreach (var measurement in template.GetMeasurements(token))
                 {
-                    measurement.IngestionTimeUtc = evt.EnqueuedTime.UtcDateTime;
-                    collector.Add((evt.PartitionId, measurement));
-                    projections++;
-                }
-                catch (Exception ex)
-                {
-                    // Translate all Normalization Mapping exceptions into a common type for easy identification.
-                    throw new NormalizationDataMappingException(ex, nameof(NormalizationDataMappingException))
-                        .AddEventContext(evt);
+                    try

```

Comment:
This will record the time taken to normalize the device event. But  are we still capturing the time it takes to produce an individual measurement?
 
Also, is `NormalizationEventConsumerService` replacing `MeasurementEventNormalizationService`?